chanacceptor/rpcacceptor.go

```diff
@@ -288,12 +325,36 @@ func (r *RPCAcceptor) sendAcceptRequests(errChan chan error,
 // validateAcceptorResponse validates the response we get from the channel
 // acceptor, returning a boolean indicating whether to accept the channel, an
 // error to send to the peer, and any validation errors that occurred.
-func validateAcceptorResponse(req lnrpc.ChannelAcceptResponse) (bool, error,
-	error) {
+func (r *RPCAcceptor) validateAcceptorResponse(dustLimit btcutil.Amount,
```

Comment:
Is there anything else we'd like to validate against here?

Reply:
All covered in terms of the requirements for BOLT2. Also added a check that we don't set > 483 for max htlcs. The other side should catch it anyway, but might as well validate early.